CONTRIBUTING.md

```diff
@@ -89,16 +89,21 @@ to make reviews and retrospection easy. Use your git commits to provide context
 for the reviewers, and the folks who will be reading the codebase in the months
 and years to come.
 
-Finalized commit messages should look similar to the following format:
+We're trying to keep all commits in `master` to follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) format.
+See [conventions](#commit-conventions) for more info on types and scopes.
+We are using squash and merge approach to PRs which means that commit descriptions are generated from PR descriptions.
 
-```text
-Short one line title
+It's recommended to use conventional commits when strarting a PR, but follow-up commits in the PR don't have to follow the convention.
```

Comment:
I think we should also mention the points under `5 Steps to Write Better Commit Messages` from this blog post https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/ somewhere here.
I really like the points where imperative mood is recommended and others.